lib/_http_outgoing.js

```diff
@@ -826,5 +831,6 @@ OutgoingMessage.prototype.pipe = function pipe() {
 };
 
 module.exports = {
-  OutgoingMessage
+  OutgoingMessage,
+  kNeedDrain
```

Comment:
We should export this from an internal module instead, otherwise there is no point to using a symbol.

Reply:
Makes sense. Resolved.